webapp-client/src/component/CategoryList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// TODO: Add support to highlight active category.
+export default class CategoryList extends React.Component {
+  // this.props.categories: a list of category ids fetched from db
+  // this.props.setContentPane: sets the current category to be displayed
```

Comment:
It would be better to validate this with proptypes. Then you can move this documentation into the proptypes object. Same for the rest of the app.

https://reactjs.org/docs/typechecking-with-proptypes.html